perf/Makefile

```diff
@@ -0,0 +1,15 @@
+#
+# Run semgrep benchmarks.
+# See './run-benchmarks --help' for options.
+#
+.PHONY: run
+run:
+	./run-benchmarks --docker returntocorp/semgrep:develop
+
+.PHONY: dummy
+dummy:
+	./run-benchmarks --dummy --docker returntocorp/semgrep:develop --upload
+
+.PHONY: clean
+clean:
+	git clean -dfX
```

Comment:
I don't like this ... because it perfoms it on the whole repo. I think I got file that I keep around deleted because
I did make clean from the wrong place one time.

Reply:
It's local, I just double-checked. Capital `X` ensures only the git-ignored files are removed, as opposed to lowercase `x` which will remove all files not tracked by git.

Reply:
yes but it removed my skip_list.txt ... which is my local gitignore